include/libtorrent/aux_/platform_util.hpp

```diff
@@ -1,11 +1,13 @@
 #ifndef TORRENT_PLATFORM_UTIL_HPP
 #define TORRENT_PLATFORM_UTIL_HPP
 
+#include "libtorrent/config.hpp"
```

Comment:
if all you need is the `TORRENT_EXTRA_EXPORT` it's sufficient to include `aux_/export.hpp`

Reply:
right, now the long wait of appveyor :)